requests-for-comments/2024-06-21-warehouse-launch-plan.md

```diff
@@ -0,0 +1,50 @@
+## [Marketing] Data warehouse launch plan
+
+The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 
+
+### Ahead of launch
+
+- [ ] Publish a Stripe data warehouse tutorial - Marketing
+- [ ] Publish a Hubspot data warehouse tutorial - Marketing
+- [ ] Publish a Postgres data warehouse tutorial - Marketing
+- [ ] Update docs as needed - Marketing
+- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - @joethreepwood
+- [x] Key art - @lottiecoxon (we have some already, open to new though)
+
+
+### Launch day activity
+
+- [ ] Publish a launch blog piece about why data warehouse is important - 
+- [ ] Update pricing page with data warehouse pricing
+- [ ] Update in-app onboarding with data warehouse option
+- [ ] Share blogpost to HackerNews - Marketing
+- [ ] Email announcement to all users - @joethreepwood
+- [ ] Social push, changelog, Pylon announcement - @joethreepwood
+
+### Follow-on activity
+
+- [ ] Source a data warehouse customer story - @joethreepwood
+- [ ] Update onboarding email flow with data warehouse options - @joethreepwood
+
+### Bigger ideas
+
+**Dashboard Templates**
+Is it possible to create dashboard templates that integrate with our integrated sources? E.g. a dashboard for getting Stripe data. 
```

Comment:
Question for @EDsCODE

Reply:
Since data warehouse insights work like normal insights, dashboard templates should work.

Reply:
Awesome. Think we can get these built ahead of launch? 

It would be awesome if we had tutorials which walked through how to create some dashboards but then also pointed to templates as an example.

Reply:
Yes